chromium/utils/translate-en-messages.py

```diff
@@ -1,139 +1,119 @@
+
+
+def overwrite_print(msg):
+    """Dynamically overwrites the current line in the terminal."""
+    stdout.write('\r' + msg.ljust(TERMINAL_WIDTH)[:TERMINAL_WIDTH])
+
+
+# Collect keys to ignore
 keys_to_ignore = []
 while True:
-    key = input('Enter key to ignore (or ENTER if done): ')
-    if not key : break
+    key = input('Enter key to ignore (or press ENTER if done): ')
+    if not key:
+        break
     keys_to_ignore.append(key)
 
-# Determine closest locales dir
-print_trunc(f'\nSearching for { locales_folder }...')
+# Locate locales directory
+print_trunc(f"\nSearching for {LOCALES_FOLDER}...")
 script_dir = os.path.abspath(os.path.dirname(__file__))
 locales_dir = None
-for root, dirs, files in os.walk(script_dir): # search script dir recursively
-    if locales_folder in dirs:
-        locales_dir = os.path.join(root, locales_folder) ; break
-else: # search script parent dirs recursively
-    parent_dir = os.path.dirname(script_dir)
-    while parent_dir and parent_dir != script_dir:
-        for root, dirs, files in os.walk(parent_dir):
-            if locales_folder in dirs:
-                locales_dir = os.path.join(root, locales_folder) ; break
-        if locales_dir : break
-        parent_dir = os.path.dirname(parent_dir)
-    else : locales_dir = None
-
-# Print result
-if locales_dir : print_trunc(f'_locales directory found!\n\n>> { locales_dir }\n')
-else : print_trunc(f'Unable to locate a { locales_folder } directory.') ; exit()
-
-# Load en/messages.json
+
+for root, dirs, _ in os.walk(script_dir):
+    if LOCALES_FOLDER in dirs:
+        locales_dir = os.path.join(root, LOCALES_FOLDER)
+        break
+
+if not locales_dir:
+    print_trunc(f"Unable to locate the {LOCALES_FOLDER} directory.")
+    exit()
+
+print_trunc(f"_locales directory found: {locales_dir}\n")
+
+# Load English messages
 msgs_filename = 'messages.json'
 en_msgs_path = os.path.join(locales_dir, 'en', msgs_filename)
+
 with open(en_msgs_path, 'r', encoding='utf-8') as en_file:
     en_messages = json.load(en_file)
 
-# Combine [target_langs] w/ languages discovered in _locales
-output_langs = list(set(target_langs)) # remove duplicates
-for root, dirs, files in os.walk(locales_dir):
+# Discover and combine languages
+output_langs = list(set(TARGET_LANGS))
+for root, dirs, _ in os.walk(locales_dir):
     for folder in dirs:
-        folder_path = os.path.join(root, folder)
-        msgs_path = os.path.join(folder_path, msgs_filename)
         discovered_lang = folder.replace('_', '-')
-        if os.path.exists(msgs_path) and discovered_lang not in output_langs : output_langs.append(discovered_lang)
-output_langs.sort() # alphabetize languages
+        if discovered_lang not in output_langs:
+            output_langs.append(discovered_lang)
+output_langs.sort()
 
-# Create/update/translate [[output_langs]/messages.json]
-langs_added, langs_skipped, langs_translated, langs_not_translated = [], [], [], []
+# Translate messages
+langs_translated = []
```

Comment:
The original comment is clearer, also the other array inits got deleted so script no longer works